bcda/api.go

```diff
@@ -200,7 +210,7 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("X-Progress", job.Status)
 		w.WriteHeader(http.StatusAccepted)
 	case "Failed":
-		writeError(fhirmodels.OperationOutcome{}, w)
+		responseutils.WriteError(&fhirmodels.OperationOutcome{}, w, http.StatusOK)
```

Comment:
Should this be OK or an error code? Based on https://github.com/smart-on-fhir/fhir-bulk-data-docs/blob/master/export.md#response---error-status-1 it seems like an error response would be correct (different from Completed but with some errors which is 200)

Reply:
Yeah I thought about this for a while. Here's my reasoning, the response for the job status itself is still a successful response, we're just returning a "failed status" for that particular job. So should this really be an error response for the polling itself? They also don't mention anything about a "fail/failure" on the bulk data status page but mix in language of "in-progress, complete, and error" which in my opinion causes this type of confusion. If an actual error happens then that will be an operation outcome (before the switch case) and if a "job failed" happens then we will have to write out the operation outcome to file(s) and list them in the response.error array while still capturing the successful jobs.

Reply:
So basically an overall job can still succeed while failing on some of the file downloads. If we actually wrote out an error response then our users would have no way of accessing the "successful" files from the output array of a complete status, and we'll essentially be throwing away all of the good files if one doesn't make it through. 1 job can be x number of files to be downloaded. Does this make sense?

Reply:
good catch, will change this. My thinking was off